doc/articles/includes/additional-macos-setup-inline.md

```diff
@@ -0,0 +1,5 @@
+In order to run Skia+GTK heads, you will need to make sure to install `gtk+3` package using the following command line, using [homebrew](https://brew.sh/):
```

Comment:
@jeromelaban not sure this is still needed for the Setup for Skia Desktop projects lately.
Is having a macOS tab still relevant here, please?
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-rider.md?plain=1#L57
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/common-issues-skia.md?plain=1#L26
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L32

Reply:
any gtk reference is not needed. If the section becomes empty, we can remove it.

Reply:
You can remove the macos tabs on these pages @LProulx86. 
Make sure to remove the tab part if there is only one tab left for some pages.

Reply:
@jeromelaban 

[uno/doc/articles/features/using-skia-gtk.md](https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L32)

I guess we keep gtk reference in this article.

Reply:
Yes, gtk for this article is required for legacy information.